Sources/SwiftKuery/ConnectionPoolConnection.swift

```diff
@@ -145,6 +179,8 @@ public class ConnectionPoolConnection: Connection {
         connection.execute(raw, parameters: parameters, onCompletion: onCompletion)
     }
     
+    // MARK: Prepared statements
+
     /// Prepare statement.
     ///
     /// - Parameter query: The query to prepare statement for.
```

Comment:
Please add the to make it - Parameter query: The query to prepare the statement for. 
(This ripples further down too, so will need changing in a few other places).

Reply:
On line 188 please add "the" to the throws description after the word prepare (there may be others of these)
"Throws: QueryError.syntaxError if query build fails, or a database error if it fails to prepare the statement."